x-pack/plugins/metrics_entities/server/services/utils/index.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+export * from './get_index_exists';
+export * from './get_transform_exists';
+export * from './compute_transform_id';
+export * from './compute_mapping_index';
```

Comment:
++ Clear pattern here on the portability of utils here. Curious if there's a core check/equivalent for `get_index_exists`.

Reply:
I don't think so. I looked around for a bit but did not see one for us.